### ISSUE TITLE
Link on Clifford panel broken, zooming not updating well on Attractors

### ISSUE THREAD

**Issue by ablythed** [pyviz-topics/examples#157]
On the [Clifford panel page](https://attractors.pyviz.demo.anaconda.com/clifford_panel), the link in the description to "Clifford attractor" gives a 404.

Also, this may just be my computer, but the final plot in the [original Attractors notebook](https://attractors-notebooks.pyviz.demo.anaconda.com/notebooks/attractors.ipynb) remains low resolution when I zoom in instead of updating to show the detailed trajectories, as below (even after waiting for 5+ minutes):

<img width="909" alt="Screen Shot 2021-06-16 at 4 23 30 PM" src="https://user-images.githubusercontent.com/71657990/122288727-ede1b680-cebf-11eb-9393-8e25194a6749.png">

**Comment by jbednar** [comment]
Thanks! It looks like the link is broken because it points to `attractors.ipynb`, which exists in the directory where clifford_panel.ipynb is maintained, but it looks like that file is not served as a notebook once it's deployed. I'm not sure why it isn't available there, but in any case, it should be safer to link to the full URL https://examples.pyviz.org/attractors anyway so that it always works, even in a deployment. Can you change clifford_panel.ipynb to replace `[Clifford attractor](attractors.ipynb)` with `[Clifford attractor](https://examples.pyviz.org/attractors)`?

For the last plot, that behavior should be expected for `dynamic=False`, which tells HoloViews to render the Datashader plot only once. I can't now remember why it says not to use a dynamic plot, but someone (maybe me!) put that there deliberately! Can you try `dynamic=True` and we can see why it wasn't left dynamic originally? Maybe it's for some long ago fixed bug, but if it's still an issue, we at least need to explain why it's there.

**Comment by ablythed** [comment]
Just changed the link. And I think this is why dynamic was set to False, because here's what happens when it's on True:

<img width="1131" alt="Screen Shot 2021-06-16 at 6 57 50 PM" src="https://user-images.githubusercontent.com/71657990/122305903-91899180-ced5-11eb-97ad-27ade90c4069.png">

**Comment by jbednar** [comment]
@jlstevens , any way around that?

**Comment by jlstevens** [comment]
> Would it have to handle streams explicitly and pass them down?

Correct. It is possible to get working but you would need to thread through the streams needed to allow dynamic zooming.

> any way around that?

As `datashade` in an operation, it should be possible to put it at the very end. i.e `datashade(dm)` on the last line where `dm` now returns the `hv.Points`. Note: I've not tested this yet but that is what I would expect!

**Comment by jbednar** [comment]
Perfect, thanks!  @ablythed , here's a replacement that works for zooming and is simpler anyway:

```python
 def hv_clif(a,b,c,d,x0=0,y0=0,n=n):
    return hv.Points(trajectory(Clifford, x0, y0, a, b, c, d, n))

x0,y0,a,b,c,d = args("Clifford")[6][1:]

dm = hv.DynamicMap(hv_clif, kdims=['a', 'b', 'c', 'd'])
dm = dm.redim.range(a=(-2.0, 2.0), b=(-2.0,2.0), c=(-2.0,2.0), d=(-2.0,2.0))
dm = dm.redim.default(a=a, b=b, c=c, d=d)

datashade(dm, cmap=inferno[::-1]).opts(width=500,height=500)
```

**Comment by ablythed** [comment]
Fixed in #163.
